docs/get-started/best-practices.md

```diff
@@ -0,0 +1,165 @@
+---
+layout: page
+title: Best Practices on Spark RAPIDS
+nav_order: 15
+parent: Getting-Started
+---
+
+# Best Practices on Spark RAPIDS
+
+This article explains the most common best practices using the RAPIDS Accelerator especially for 
+performance tuning and troubleshooting.
+
+## Workload Qualification
+
+By following [Workload Qualification](./getting-started-workload-qualification.md), you can identify
+the best candidate Spark applications for the RAPIDS Accelerator and also the feature gaps.
+
+Based on [Qualification tool](../spark-qualification-tool.md)'s output, you can start with the top N
+recommended CPU Spark jobs, especially if those jobs are computation heavy jobs (Large Joins, 
+HashAggregate, Windowing, Sorting).
+
+After those candidate jobs are actually run on GPU using the RAPIDS Accelerator, check Spark driver 
+log to find the not-supported messages starting with `!`. Some missing features might be enabled by
+turning on some config, but please firstly understand this config's limitation based on 
+[Configuration](../configs.md). For other unsupported features, please file feature request on 
+[spark-rapids github repo](https://github.com/NVIDIA/spark-rapids/issues) with a minimum reproduce
+and the not-supported messages if they are non-senstive data.
+
+## Performance Tuning
```

Comment:
We already have a tuning guide, and I'd rather not scatter tuning suggestions across multiple documents.  I recommend moving this section to the tuning guide and referencing it from here.

Reply:
@jlowe I think one purpose of this best practice guide is to give user a simplified set of tips instead of digging into the complete tuning guide. How do you think?

Reply:
Having a best practices doc is great, but I don't see that as a necessity to spread tuning information in multiple places.  What I'm suggesting is to move the best practices related to tuning (not OOM debugging or other stuff) to the tuning guide, since many people will reach for the tuning guide and miss stuff put here.  Main point is I don't want N places where users should look for tuning tips, as we already have a place for tuning tips.

Reply:
@jlowe  sure just added the tuning guide. How is now?

Reply:
Glad to see the tuning tips moved to where we already have other tips.  We may want to do a cleanup/reorg pass on the tuning doc to move the more common/important tips to the top of the document or in its own section, but we can do that in a followup issue if needed.

Reply:
Sure, that is a good idea.